common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -117,6 +117,9 @@
 new_contract('BlockKey', BlockKey)
 new_contract('XBlock', XBlock)
 
+# Get detached categories.
+_DETACHED_CATEGORIES = [name for name, __ in XBlock.load_tagged_classes("detached")]
```

Comment:
This same line exists here: https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/mongo/base.py#L80

Rather than duplicating this line, can we import it from a common source?